app/system/survey/js/prompts.js

```diff
@@ -2152,18 +2171,39 @@ promptTypes.birthdate = promptTypes.date.extend({
         }
     },
 });
+promptTypes.date_no_time = promptTypes.base_date.extend({
+    type: "date_no_time",
+    showTime: false,
+    timeFormat: "MM/DD/YYYY",
```

Comment:
timeFormat is still backwards but it's supposed to be unused, but by leaving it in local format could that cause problems?

Reply:
This is the display format, not the saved format. Do you want me to change that behavior?